rasa/shared/nlu/training_data/training_data.py

```diff
@@ -884,6 +970,18 @@ def __init__(self, *args, **kwargs):
         )
 
 
+RELEVANT_MESSAGE_KEYS = [
```

Comment:
Can we move this to the constants file?

Reply:
moved it inside `TrainingDataChunk` class